app/views/News.js

```diff
@@ -129,14 +102,33 @@ class NewsScreen extends Component {
                 flex: 1,
                 paddingVertical: 16,
               }}>
-              <Carousel
-                data={this.state.newsUrls}
-                renderItem={this.renderItem}
-                sliderWidth={width}
-                itemWidth={width * 0.85}
-                layout={'default'}
-                scrollEnabled
-              />
+              <ScrollView style={{ width: width * 0.85, alignSelf: 'center' }}>
```

Comment:
Changing out Carousel for a ScrollView seems like a big UX change. Can you provide a video of how this works now?

Reply:
Sure! [Scrolling](https://youtu.be/US53QQS8kw0)

Reply:
It is much more user-friendly now in terms of scrolling.

Reply:
Made the UI a bit more proportional for small and large screens
![Nexus4](https://user-images.githubusercontent.com/43054656/81997759-548a0600-9616-11ea-8c0c-a114e0e5069d.png)
![Pixel3XL](https://user-images.githubusercontent.com/43054656/81997768-5a7fe700-9616-11ea-8f81-d153ce0f3048.png)